hw/ip/rv_plic/fpv/rv_plic_example_fpv.core

```diff
@@ -2,8 +2,8 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:fpv:rv_plic_fpv:0.1"
```

Comment:
We've been using 'generic' to describe this: see `hw/ip/tlul/generic_dv`.

Reply:
sure, I will update the naming. Thank you for the suggestion.